### PR TITLE
doc: worker_manager use suggestion to reduce boilerplate

### PR DESCRIPTION
**Pull request by machinescream**
_Description of what this PR is changing or adding, and why:
To recommend a good library for managing long-lived isolates and providing the capability to cancel heavy tasks when they are no longer needed.

_Issues fixed by this PR (if any):_

## Presubmit checklist

- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: src/perf/isolates.md [hunk 1]
```diff
@@ -209,6 +209,8 @@ needs to be run repeatedly throughout the lifetime of your application,
 or if you have a process that runs over a period of time
 and needs to yield multiple return values to the main isolate.
 
+Or, you can try to use [worker_manager](https://pub.dev/packages/worker_manager) to simplify your long-lived isolates managing.
```
Comment (sfshaza2) [inline src/perf/isolates.md:212]:
```suggestion
Or, you might use [worker_manager][] to manage long-lived isolates.

[worker_manager]: {{site.pub-pkg}}/worker_manager
```

**Review by sfshaza2 — APPROVED** [review]
lgtm!

**Comment by sfshaza2** [comment]
(I ran this past several DREs...)

**Review by domesticmouse — APPROVED** [review]

**Comment by machinescream** [comment]
@sfshaza2 Should I do something to pass flutter-website-staging job?

**Comment by sfshaza2** [comment]
Nope, you passed it!

**Pull request merged**